types/function_list_builder.go

```diff
@@ -51,19 +53,47 @@ func (s *FunctionLookupBuilder) Build() (map[string][]string, error) {
 		return serviceMap, marshalErr
 	}
 
+	topicDelim := ","
+
+	if delim, exists := os.LookupEnv("topic_delimiter"); exists && len(delim) > 0 {
```

Comment:
The connector-sdk is consumed by other go programs so can you pass in this delimitator via constructor?

Reply:
@rgee0

Reply:
Addressed this